yotta/build.py

```diff
@@ -29,6 +29,7 @@ def addOptions(parser, add_build_targets=True):
     )
     parser.add_argument('-r', '--release-build', dest='release_build', action='store_true', default=True)
     parser.add_argument('-d', '--debug-build', dest='release_build', action='store_false', default=True)
+    parser.add_argument('-s', '--strip', dest='strip', action='store_true', default=False)
```

Comment:
Is `--strip` the best switch name? perhaps `--release-no-debug-build` or something better inline with current naming?